test/integration/storage/test_objects.rb

```diff
@@ -0,0 +1,114 @@
+require "helpers/integration_test_helper"
+
+class TestObjects < FogIntegrationTest
+  def setup
+    # Uncomment this if you want to make real requests to GCE (you _will_ be billed!)
```

Comment:
I'd defer to you on this. If there's somewhere higher up to put this warning it sounds like the right thing to do to me. These tests could use some work, and I'm not familiar enough with the stack for `rake test` to suggest a better place to include this warning (although I'm sure there is one).